mathesar/api/serializers/columns.py

```diff
@@ -35,13 +37,23 @@ def run_validation(self, data=empty):
         return super(TypeOptionSerializer, self).run_validation(data)
 
 
-class SimpleColumnSerializer(serializers.Serializer):
+class SimpleColumnSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Column
+        fields = ('id', 'name', 'type', 'type_options')
     name = serializers.CharField()
     type = serializers.CharField(source='plain_type')
     type_options = TypeOptionSerializer(required=False, allow_null=True)
 
 
 class ColumnSerializer(SimpleColumnSerializer):
+    class Meta(SimpleColumnSerializer.Meta):
+        fields = SimpleColumnSerializer.Meta.fields + ('nullable', 'primary_key', 'source_column', 'copy_source_data',
```

Comment:
I find lists and tuples easier to read when they're one item per line. I would do something like this:

```python
fields = SimpleColumnSerializer.Meta.fields + (
   'nullable',
   'primary_key',
   'source_column',
   'copy_source_data',
   etc...
)
```